File: frontend_refactor/src/dtos/stock/StockLog.response.dto.ts
//src/dtos/stock/StockLog.response.dto.ts


import { StockActionType } from "@/apis/enums/StockActionType";

export interface StockLogResponseDto{
  stockLogId : number;
  type : StockActionType;
  employeName : string;
  bookTitle : string;
  branchName : string;
  amount : number;
  bookAmount : number;
  actionDate: string;
  description : string;
}

export interface StockLogDetailResponseDto{
  stockLogId : number;
  type : StockActionType;
  employeName : string;
  bookTitle : string;
  branchName : string;
  amount : number;
  bookAmount : number;
  actionDate: string;
  description : string;
}

File: backend/src/main/java/com/bookhub/bookhub_back/dto/stock/response/StockLogResponseDto.java
package com.bookhub.bookhub_back.dto.stock.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Getter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class StockLogResponseDto {
    private Long stockLogId;
    private String type;
    private String employeeName;
    private String bookTitle;
    private String branchName;
    private Long amount;
    private LocalDate actionDate;
}
